### ISSUE TITLE
Missing documentation on how to install the cilk library dependancy

### ISSUE THREAD

**Issue by krame505** [melt-umn/ableC-cilk#2]
There should probably be a blurb about this in the README - I've been having trouble figuring out how to install this library.

**Comment by remexre** [comment]
Agreed that there should be a blurb, but does https://github.umn.edu/melt/cilk work for you? Alternatively, on my system `libcilkrts.so` is provided by the `gcc-libs`, so it might already be installed.

**Comment by krame505** [comment]
Aha, thanks!  Yeah, a link to this repo and a brief explanation would do.

Lucas Kramer

**Comment by krame505** [comment]
Actually that is on the private github.umn.edu melt site, so we need to either migrate over that repo or do something else.
